src/__tests__/config.test.ts

```diff
@@ -70,130 +56,10 @@ describe('validateConfiguration', () => {
 });
 
 describe('getProjectConfigSchema', () => {
-  test('returns non-empty object', async () => {
+  test('returns non-empty object', () => {
     const projectConfigSchema = getProjectConfigSchema();
 
-    expect(projectConfigSchema).toBeTruthy();
```

Comment:
Took this out because even an empty object is truthy. (Why, JS, why??) Also, if the next to assertions pass, it's clearly not empty in any case.